tests/explorer.api.tests/ExplorationTestScope.cs

```diff
@@ -17,70 +17,126 @@ public class ExplorationTestScope : IDisposable
     {
         private const string TestApiUrl = "https://attack.aircloak.com/api/";
         private bool disposedValue;
+        private readonly Container rootContainer;
 
         public ExplorationTestScope(Container rootContainer)
         {
-            Scope = rootContainer.GetNestedContainer();
+            this.rootContainer = rootContainer;
         }
 
-        public INestedContainer Scope { get; }
+        public (string, Cassette)? LoadedCassette { get; private set; }
```

Comment:
There is no place where you use the first item from the tuple, so it seems possible to drop it.